websites/B/BloomHost/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+		clientId: "873728851052752896"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ hostname, pathname } = document.location;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "bloom_logo",
+		startTimestamp: browsingTimestamp
+	};
+	if (hostname === "bloom.host") {
+		presenceData.smallImageKey = "globe-solid";
+		presenceData.details = "Viewing the main website";
+		if (pathname.includes("minecraft"))
+			presenceData.state = "Reading: Minecraft Plans";
+
+		if (pathname.includes("vps")) presenceData.state = "Reading: VPS Plans";
+
+		if (pathname.includes("duck-panel"))
+			presenceData.state = "Reading: About the Duck Panel";
+
+		if (pathname.includes("about-bloom"))
+			presenceData.state = "Reading: About Bloom";
```

Comment:
these won't all be true at the same time, they should be else ifs or switch statements

Reply:
Else ifs  will just give me a linting error, I will use switch statements instead